app/src/main/java/com/example/playme/FBReg.kt

```diff
@@ -0,0 +1,160 @@
+            .fillMaxSize()
+            .background(Color.Black),
+        contentAlignment = Alignment.Center
+    ) {
+        Column(
+            verticalArrangement = Arrangement.Center,
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            Text(text = "PlayMe",
+                modifier = Modifier.padding(16.dp,16.dp,16.dp,100.dp),
+                fontSize = 35.sp,
+                fontFamily = customFontFamily,
+                color = Color.Yellow
+            )
+
+            TextField(
+                value = email,
+                onValueChange = { email = it },
+                label = { Text("Електронна адреса") },
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(16.dp),
+                colors = TextFieldDefaults.textFieldColors(
+                    textColor = Color.White,
+                    focusedIndicatorColor = Color.White,
+                    unfocusedIndicatorColor = Color.White
+                ),
+                textStyle = MaterialTheme.typography.bodySmall.copy(
+                    fontFamily = customFontFamily,
+                    fontSize = 16.sp,
+                    color = Color.DarkGray
+                )
+            )
+
+            Spacer(modifier = Modifier.height(16.dp))
+
+            TextField(
+                value = password,
+                onValueChange = { password = it },
+                label = { Text("Пароль") },
+                visualTransformation = PasswordVisualTransformation(),
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(16.dp),
+                colors = TextFieldDefaults.textFieldColors(
+                    textColor = Color.White,
+                    focusedIndicatorColor = Color.White,
+                    unfocusedIndicatorColor = Color.White
+                ),
+                textStyle = MaterialTheme.typography.bodySmall.copy(
+                    fontFamily = customFontFamily,
+                    fontSize = 16.sp,
+                    color = Color.DarkGray
+                )
+            )
+
+            Spacer(modifier = Modifier.height(16.dp))
+
+            TextField(
+                value = confirmPassword,
+                onValueChange = { confirmPassword = it },
+                label = { Text("Підтвердіть пароль") },
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(16.dp),
+                colors = TextFieldDefaults.textFieldColors(
+                    textColor = Color.White,
+                    focusedIndicatorColor = Color.White,
+                    unfocusedIndicatorColor = Color.White
+                ),
+                textStyle = MaterialTheme.typography.bodySmall.copy(
+                    fontFamily = customFontFamily,
+                    fontSize = 16.sp,
+                    color = Color.DarkGray
+                )
+            )
```

Comment:
Don't repeat yourself, create TextField extended class with variables to write placed in constructor, if need change basic implementation of new class override methods/properties